airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/models/airbyte_protocol.py

```diff
@@ -14,51 +14,51 @@
 
 
 class Type(Enum):
-    RECORD = "RECORD"
-    STATE = "STATE"
-    LOG = "LOG"
-    SPEC = "SPEC"
-    CONNECTION_STATUS = "CONNECTION_STATUS"
-    CATALOG = "CATALOG"
+    RECORD = 'RECORD'
+    STATE = 'STATE'
+    LOG = 'LOG'
+    SPEC = 'SPEC'
+    CONNECTION_STATUS = 'CONNECTION_STATUS'
+    CATALOG = 'CATALOG'
```

Comment:
Nitpick. I think if you run `format`, the quotation mark changes in this file will go away.